acceptance/nodes/redhat7-64m-64a.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Should this file live under `.../acceptance/config/nodes` instead of this directory?  That's the only other one like this I found in the puppet-server repo.